Documentation/Signoffs/detaildesign_camera.md

```diff
@@ -0,0 +1,97 @@
+# Detailed Design for Camera Subsystem within Primary Data Acquisition System
+
+## Big Picture
+The primary data acquisition system will utilize real-time camera sensors to collect images of sectors of parking spots in the lot behind Bell Hall, in which these images will then be transmitted via wireless communication to a server where the images will be processed by an artificial intelligence (AI) algorithm to determine the number of red parking spots that are filled and unfilled by cars. This subsystem is a critical subsystem because these cameras will provide the primary data to the system that will determine the count of vehicles in a lot.
+
+## Specifications
```

Comment:
This is not the appropriate place to discuss the chosen camera's abilities. First, establish what is required and what constraints you have have to abide by, then show that the camera is capable of meeting those requirements.